cupy/random/generator.py

```diff
@@ -100,6 +101,59 @@ def lognormal(self, mean=0.0, sigma=1.0, size=None, dtype=float):
             func = curand.generateLogNormalDouble
         return self._generate_normal(func, size, dtype, mean, sigma)
 
+    def multivariate_normal(self, mean, cov, size=None, check_valid='ignore',
+                            tol=1e-8, dtype=float):
+        """Returns an array of samples drawn from the multivariate normal distribution.
+
+        .. seealso::
+            :func:`cupy.random.multivariate_normal` for full documentation,
+            :meth:`numpy.random.RandomState.multivariate_normal`
+        """
+        mean = cupy.asarray(mean, dtype=dtype)
+        cov = cupy.asarray(cov, dtype=dtype)
+        if size is None:
+            shape = []
+        elif isinstance(size, (int, cupy.integer)):
+            shape = [size]
+        else:
+            shape = size
+
+        if mean.ndim != 1:
+            raise ValueError("mean must be 1 dimensional")
+        if (cov.ndim != 2) or (cov.shape[0] != cov.shape[1]):
+            raise ValueError("cov must be 2 dimensional and square")
+        if len(mean) != len(cov):
+            raise ValueError("mean and cov must have same length")
+        final_shape = list(shape)
+        final_shape.append(len(mean))
+
+        x = self.standard_normal(size=final_shape, dtype=dtype)
+        x = x.reshape(-1, len(mean))
```

Comment:
Why is this line needed?

Reply:
This code use numpy random code.(https://github.com/numpy/numpy/blob/18f2385b29bdd62701a1a82d7bf33fd87430a05e/numpy/random/mtrand/mtrand.pyx)
But, I think this line is not needed. So I'll erase.